package.json

```diff
@@ -18,34 +18,38 @@
     },
     "dependencies": {
         "@nuxtjs/axios": "^5.13.6",
-        "@nuxtjs/composition-api": "^0.34.0",
         "@nuxtjs/dotenv": "^1.4.2",
-        "@pinia/nuxt": "^0.2.1",
-        "bootstrap": "^4.6.2",
-        "bootstrap-vue": "^2.23.1",
+        "@pinia/nuxt": "^0.5.2",
+        "@popperjs/core": "^2.11.8",
+        "@rollup/plugin-json": "4.1.0",
+        "@rollup/pluginutils": "3.1.0",
```

Comment:
why do we need these ?

Reply:
i think the build fails without them, maybe they shouldn't be dependencies but 'resolution'